actionpack/lib/action_dispatch/middleware/cookies.rb

```diff
@@ -534,6 +543,10 @@ def digest
         def key_generator
           request.key_generator
         end
+
+        def same_site_protection
```

Comment:
If you're not calling it anywhere directly, then I'd say to not add the method right now. If it's needed in the future, it's easy to add :)